src/System.Net.Requests/tests/HttpWebRequestHeaderTest.cs

```diff
@@ -102,5 +101,62 @@ public void HttpWebRequest_EndGetRequestStreamContext_Null(Uri remoteServer)
                 Assert.Equal(null, context);
             }
         }
+
+        [OuterLoop]
+        [Theory, MemberData(nameof(EchoServers))]
+        public void HttpWebRequest_Connection_Validate(Uri remoteServer)
+        {
+            HttpWebRequest request = WebRequest.CreateHttp(remoteServer);
+            Assert.Throws<ArgumentException>(() => { request.Connection = "Close;keep-alive"; });
+        }
+
+        [OuterLoop]
+        [Theory, MemberData(nameof(EchoServers))]
+        public void HttpWebRequest_DefaultCachePolicy_Validate(Uri remoteServer)
+        {
+            Assert.Equal(Cache.RequestCacheLevel.BypassCache, HttpWebRequest.DefaultCachePolicy.Level);
+        }
+
+        [OuterLoop]
+        [Theory, MemberData(nameof(EchoServers))]
+        public void HttpWebRequest_Proxy_Validate(Uri remoteServer)
+        {
+            HttpWebRequest request = WebRequest.CreateHttp(remoteServer);
+            using (HttpWebResponse response = (HttpWebResponse)request.GetResponse())
+            {
+                Assert.Throws<InvalidOperationException>(() => { request.Proxy = WebRequest.DefaultWebProxy; });
+            }
+        }
+
+        [OuterLoop]
+        [Theory, MemberData(nameof(EchoServers))]
+        public void HttpWebRequest_TimeOut_Validate(Uri remoteServer)
+        {
+            HttpWebRequest request = WebRequest.CreateHttp(remoteServer);
+            Assert.Throws<ArgumentOutOfRangeException>(() => { request.Timeout = -1; });
+            request.Timeout = 100;
```

Comment:
Is there a test for verifying the behavior of the timeout, i.e. that it causes the same exception to be thrown as on desktop when cancellation occurs?  Based on a quick look at the implementation, I suspect we're going to see different behavior.